### ISSUE TITLE
Presentation peer review

### ISSUE THREAD

**Issue by Priyarsha31** [INFO526-DataViz/project-final-Graph-Gurus#8]
The following is the peer review of the Presentation proposal by Plot Pioneers. The team members that participated in this review are 

- Athira Thekkemdathil Anilkumar - @AthiraThekkemadathilAnilkumar 
- Priyarsha Vallabhaneni - @Priyarsha31 
- Shalon Walter - @shalonwalter 
- Chethana Thondebhavi Shantha Kumar - @Chethana-16

- Content: Is the research question well designed and is the data being used relevant to the research question?
Yes, the research questions were well framed and implemented to get all the desired results answering them. 

- Content: Did the team use appropriate visualizations and did they interpret them accurately?
Yes, the visualizations were apt. They interpreted them accurately but missed to interpret one plot overall, which is fine.

- Creativity and Critical Thought: Is the project carefully thought out? Are the limitations carefully considered? Does it appear that time and effort went into the planning and implementation of the project?
Yes, the project was well selected, despite knowing the limitations they did try to overcome most of them. Also, the app was well designed. Although we had slight confusion whether it's a dashboard or a shiny app.

- Slides: Are the slides well organized, readable, not full of text, featuring figures with legible labels, legends, etc.?
All the slides were well arranged, easy to read and not too wordy. Also the figures were labeled appropriately.

- Professionalism: How well did the team present? Does the presentation appear to be well practiced? Are they reading off of a script? Did everyone get a chance to say something meaningful about the project?
 The presentation is well prepared and presented. Everyone did equally contribute to the overall presentation.

**Comment by SwethaSiripurapuu** [comment]
Thank you!!
